src/d3d12/d3d12_functions.hpp

```diff
@@ -108,8 +108,12 @@ namespace wr::d3d12
 
 	// Staging Buffer
 	[[nodiscard]] StagingBuffer* CreateStagingBuffer(Device* device, void* data, std::uint64_t size, std::uint64_t stride, ResourceState resource_state);
+	void SetStagingBufferData(StagingBuffer* buffer, void* data, std::uint64_t size, std::uint64_t offset);
```

Comment:
Doesn't seem to be consistent naming. Other functions are called `UpdateXXX` IIRC.